test_pii_data.py

```diff
@@ -176,28 +176,32 @@ def test_has_ipv4(self):
     def test_has_ipv6(self):
         test_data = Pii('2001:0db8:85a3:0000:0000:8a2e:0370:7334')
         self.assertTrue(test_data.has_ipv6())  # test a valid address
+        print("\n" + test_data.has_ipv6(True))
```

Comment:
This print does not actually test the anonymize functionality.  You want to do something like this:
```
        print("\n" + test_data.has_ipv6(True))
        self.assertEqual('[iPv6 address]',
                         test_data.has_ipv6(True))
```